src/Security/Member.php

```diff
@@ -382,7 +382,7 @@ public function canLogIn()
      *
      * @return bool
      */
-    public function isLockedOut()
+    protected function isLockedOut()
```

Comment:
That comment doesn't necessarily mandate visibility because public API is used internally (which is a good idea in general)